neo/test/iotest/test_proxyobjects.py

```diff
@@ -54,8 +54,8 @@ def test_AnalogSignalProxy(self):
         # ceil next sample when slicing
         anasig = proxy_anasig.load(time_slice=(1.99999 * pq.s, 5.000001 * pq.s))
         assert anasig.t_start == 2. * pq.s
-        assert anasig.duration == 3. * pq.s
-        assert anasig.shape == (30000, 8)
+        assert anasig.duration == 3.0001 * pq.s
+        assert anasig.shape == (30001, 8)
```

Comment:
I think theses test were design with np.round in mind.

Reply:
Reverted the tests and switched to rint